main/src/ui/res/layout/allowed_application_layout.xml

```diff
@@ -22,35 +22,51 @@
         android:layout_height="@android:dimen/app_icon_size"
         android:layout_rowSpan="1"
         android:layout_marginEnd="8dip"
+        android:contentDescription="@null"
         android:scaleType="centerInside"
-        tools:background="@drawable/icon"
-        android:contentDescription="@null" />
+        tools:background="@drawable/icon" />
 
     <TextView
-        tools:text="@string/app"
         android:id="@+id/app_name"
-        android:layout_width="0dip"
-        android:layout_columnSpan="2"
+        android:layout_width="101dp"
         android:layout_rowSpan="1"
+        android:layout_columnSpan="2"
         android:layout_gravity="fill_horizontal|center_vertical"
         android:layout_marginTop="2dip"
-        android:singleLine="true"
         android:ellipsize="marquee"
+        android:singleLine="true"
+        android:textAlignment="viewStart"
         android:textAppearance="?android:attr/textAppearanceMedium"
-        android:textAlignment="viewStart" />
+        tools:text="@string/app" />
 
-    <CheckBox android:id="@+id/app_selected"
-        android:layout_marginStart="8dip"
+    <CheckBox
+        android:id="@+id/app_selected"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:layout_rowSpan="1"
         android:layout_gravity="center_vertical"
+        android:layout_marginStart="8dip"
+        android:minWidth="48dp"
+        android:minHeight="48dp"
+        android:visibility="visible" />
+
+    <CheckBox
+        android:id="@+id/app_selected_mobile"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
         android:layout_rowSpan="1"
+        android:layout_gravity="center_vertical"
+        android:layout_marginStart="8dip"
+        android:minWidth="48dp"
+        android:minHeight="48dp"
         android:visibility="visible" />
```

Comment:
This adds a second checkbox without any UI indication what it does.

Reply:
Yes, what do you think about if the explanation would be above e.g. with an icon/text.